Application/mapper/TestIdsMapper.go

```diff
@@ -0,0 +1,34 @@
+package mapper
+
+import (
+	"github.com/apmath-web/expenses/Application/viewModels"
+	"github.com/apmath-web/expenses/Domain/models"
+	"math/rand"
+	"testing"
+)
+
+func TestIdsViewMapper(t *testing.T) {
+	id := rand.Int()
+	idSlice := []int{rand.Int(), rand.Int(), rand.Int()}
+
+	vm := viewModels.IdsViewModel{}
```

Comment:
```suggestion
	vm := viewModels.IdsViewModel{id, idSlice}
```
Ис троки ниже переименовать стоит